core/chains/evm/config/toml/defaults/Hedera_Testnet.toml

```diff
@@ -0,0 +1,33 @@
+ChainID = '296'
+ChainType = 'hedera'
+# considering the 3-5 (6 including a buffer) seconds of finality and 2 seconds block production
+# we set the depth to 6/2 = 3 blocks, setting to 10 for safety
+FinalityDepth = 10
+#Hedera has high TPS, so polling less often 
+LogPollInterval = '10s'
+MinIncomingConfirmations = 1
+#turning off the liveness detection, since the Websocket connection is not reliable, can be turned on if needed
```

Comment:
Please update all the comments with consistent formatting (space and capitalized):
```suggestion
# Hedera has high TPS, so polling less often 
LogPollInterval = '10s'
MinIncomingConfirmations = 1
# Turning off the liveness detection, since the Websocket connection is not reliable, can be turned on if needed
```

Reply:
Thanks for pointing it, not sure how I missed it 🤦🏽